corehq/apps/api/tests.py

```diff
@@ -1667,6 +1675,46 @@ def test_basic(self):
         self.assert_accessible_via_sessions(self.list_endpoint)
 
 
+class AdminAPITest(APIResourceTest):
+    api_name = 'global'
+
+    @classmethod
+    def setUpClass(cls):
+        initialize_index_and_mapping(get_es_new(), DOMAIN_INDEX_INFO)
+        super(AdminAPITest, cls).setUpClass()
+
+    def assert_admin_resource(self, url):
+        # normal user can't access admin resource
+        response = self._assert_auth_get_resource(url)
+        self.assertEqual(response.status_code, 401)
+        # admin user should be able to access
+        response = self._assert_auth_get_resource(url, username='admin', password='pass')
+        self.assertEqual(response.status_code, 200)
+
+    def test_basic(self):
+        community_domain = Domain.get_or_create_with_name('dm', is_active=True)
+        new_user = WebUser.create(community_domain.name, 'admin', 'pass', is_superuser=True)
+        new_user.save()
+
+        self.addCleanup(community_domain.delete)
+        self.addCleanup(new_user.delete)
+
+        for resource in ADMIN_API_LIST:
+            if resource is accounting.DefaultProductPlanResource:
+                # this reource has a bug
```

Comment:
nit: this comment would probably be more useful if it explained a bit more about the bug